src/pages/Regions/RegionAdministration.tsx

```diff
@@ -196,18 +207,15 @@ const RegionAdministration = () => {
         </Button>
       </div>
 
-      <Modal
-        title="Редагувати діловода"
-        visible={visibleModal}
-        onOk={handleOk}
-        onCancel={handleOk}
-        footer={null}
-      >
-        <AddNewSecretaryForm
-          onAdd={handleOk}
-          admin={admin} 
-        ></AddNewSecretaryForm>
-      </Modal>
+          <AddAdministratorModal
+            admin={admin}
+            setAdmin={setAdmin}
+            visibleModal={visibleModal}
+            setVisibleModal={setVisibleModal}
+            regionId={+id}
+            regionName={region}
+            onAdd={onAdd}
+          ></AddAdministratorModal>
```

Comment:
you can simply write "/>" instead of "></AddAdministratorModal>"

Reply:
Fixed